tests/sourceMatch.py

```diff
@@ -50,10 +50,9 @@ class SourceMatchTestCase(unittest.TestCase):
 
     def setUp(self):
         schema = afwTable.SourceTable.makeMinimalSchema()
-        schema.setVersion(0)
-        fluxKey = schema.addField("flux", type=float)
-        fluxErrKey = schema.addField("flux.err", type=float)
-        fluxFlagKey = schema.addField("flux.flags", type="Flag")
+        fluxKey = schema.addField("flux_flux", type=float)
+        fluxErrKey = schema.addField("flux_sigma", type=float)
+        fluxFlagKey = schema.addField("flux_flag", type="Flag")
         self.table = afwTable.SourceTable.make(schema)
```

Comment:
According to https://confluence.lsstcorp.org/display/DM/Measurement+Framework+Overhaul+Release+Notes:
Uncertainties on individual fields should be saved with names that end in "_<p>Sigma", where "<p>" is the suffix of the field the uncertainty corresponds to (e.g. "_fluxSigma" or "_xSigma").